.bazelrc

```diff
@@ -31,6 +31,9 @@ build --action_env=PATH --host_action_env=PATH
 build --enable_platform_specific_config
 build --test_summary=terse
 
+# TODO(keith): Remove once this is the default
+build --incompatible_config_setting_private_default_visibility
```

Comment:
You probably have to enable `--incompatible_enforce_config_setting_visibility` as well to make sure this actually take effect.

Reply:
https://github.com/envoyproxy/envoy/pull/24270